project/profiles/capacity4more/behat/features/bootstrap/FeatureContext/Group.php

```diff
@@ -289,9 +289,6 @@ public function aGroupWithAccessIsCreatedWithGroupManager($title, $access, $user
     // This is the required banner
     $steps[] = new Step\When('I attach the file to the field banner');
 
-    // This is the required message to admin.
-    $steps[] = new Step\When('I fill in "edit-field-message-to-site-admin-und-0-value" with "This is default message to admin."');
```

Comment:
@ordavidil 
Is it necessary to remove this line? 
This field is required during creation of the group.